### ISSUE TITLE
[iPad/Mac]a11y_Datagrid> controlled-selectable> A11y> 1 violations & 1 incomplete is appearing

### ISSUE THREAD

**Issue by Uditsingh67** [cengage/react-magma#878]
**Describe the bug**
Datagrid>> "1 violations & 1 incomplete" is appearing for "controlled-selectable".

**To Reproduce**
**Steps to reproduce the behavior:**

1. Go to 'https://storybook-preview-dev--upbeat-sinoussi-f675aa.netlify.app/?path=/story/datagrid--controlled-selectable
2. Click on 'Accessibility" add on
3. "1 violations & 1 incomplete " is appearing

Expected behavior
"0 violations & 0 incomplete" should appear for datagrid> controlled-selectable.

<html>
<body>

OS | Version | Result
-- | -- | --
Win 10 (Chrome) | 102.0.5005.63 | (http://somup.com/c3jbiRu93S)
Win 10 (Firefox) | 101.0 | (http://somup.com/c3jbjnu90X)
Win 10 (Edge) | 103.0.1264.37 | (http://somup.com/c3jbjfu90F)
MAC (safari) | 15.3 | (http://somup.com/c3jbjcu901)
Ipad (safari) | v15.1 | (http://somup.com/c3jbiLu93C)

</body>
</html>

**Comment by Uditsingh67** [comment]
The issue is still reproducible.
See Screen-cast : http://somup.com/c3jZjiUhbH
Thanks !

**Comment by silvalaura** [comment]
Note: reproducible only on Safari

**Comment by Uditsingh67** [comment]
Issue is reproducible only on safari.
See Screen-casts: 
Safari : (http://somup.com/c36u1FvxQE)
Chrome : (http://somup.com/c36u1Tvx6f)
